instrumentation-core/spring/spring-webmvc-3.1-core/spring-webmvc-3.1-core.gradle

```diff
@@ -0,0 +1,52 @@
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_7
+}
+
+group = 'io.opentelemetry.instrumentation'
+version = '0.0.1-SNAPSHOT'
+
+apply from: "$rootDir/gradle/java.gradle"
+apply plugin: 'muzzle'
+
+muzzle {
+  pass {
+    group = 'org.springframework'
+    module = 'spring-webmvc'
+    versions = "[3.1.0.RELEASE,]"
+    skipVersions += ['1.2.1', '1.2.2', '1.2.3', '1.2.4'] // broken releases... missing dependencies
+    skipVersions += '3.2.1.RELEASE' // missing a required class.  (bad release?)
+    extraDependency "javax.servlet:javax.servlet-api:3.0.1"
+    assertInverse = true
+  }
+
+  // FIXME: webmvc depends on web, so we need a separate integration for spring-web specifically.
+  fail {
+    group = 'org.springframework'
+    module = 'spring-web'
+    versions = "[,]"
+    skipVersions += ['1.2.1', '1.2.2', '1.2.3', '1.2.4'] // broken releases... missing dependencies
+    extraDependency "javax.servlet:javax.servlet-api:3.0.1"
+  }
+}
+
```

Comment:
Moved muzzle condition from `instrumentation/spring-webmvc-3.1/spring-webmvc-3.1.gradle` to `instrumentation-core/spring/spring-webmvc-3.1-core/spring-webmvc-3.1-core.gradle`